companion/src/modeledit/customfunctions.cpp

```diff
@@ -344,10 +346,11 @@ void CustomFunctionsPanel::refreshCustomFunction(int i, bool modified)
   else {
     fswtchSwtch[i]->setCurrentIndex(fswtchSwtch[i]->findData(cfn.swtch.toValue()));
     fswtchFunc[i]->setCurrentIndex(fswtchFunc[i]->findData(cfn.func));
+    fswtchEnable[i]->setChecked(cfn.enabled);
   }
 
   if (!cfn.isEmpty()) {
-    widgetsMask |= CUSTOM_FUNCTION_SHOW_FUNC;
+    widgetsMask |= CUSTOM_FUNCTION_SHOW_FUNC | CUSTOM_FUNCTION_ENABLE;
```

Comment:
Since this proposed to be on on all functions there is no need for it to be in the mask which gives the impression in other areas of the code that it is conditional.

Reply:
This looks like the most natural way to control visibility (the checkbox must be hidden for empty functions)